examples/dfs_bve_rh54.cpp

```diff
@@ -163,7 +164,7 @@ int main(int argc, char* argv[]) {
 
     // timestepping loop
     for (int time_idx=0; time_idx<nsteps; ++time_idx) {
-      sphere.advance_timestep(rk2_solver);
+      sphere.advance_timestep(rk4_solver);
```

Comment:
Nice work! Now you can simply add 

```
// using SolverType = DFS::DFSRK2<seed_type>;
using SolverType = DFS::DFSRK4<seed_type>;
```

to the top of the program (but after `seed_type` is defined) so that you only have to change 1 line to change solvers.

Reply:
Then the solver declaration line (143) would be `SolverType solver(dt, sphere);` and the timestepping loop would just call `sphere.advance_timestep(solver);`

Then, to change solver types, you'd only have to change that one `using` statement, above.